ariac-server/ariac-server/run_ariac_task.sh

```diff
@@ -20,20 +20,6 @@ usage()
   exit 1
 }
 
-# Wait until the /ariac/competition_state returns "data: done".
```

Comment:
@caguero I modified the script from https://bitbucket.org/osrf/ariac/pull-requests/58 to use the `ARIAC_EXIT_ON_COMPLETION` var instead of checking for 'done'. This should bring the simulation down when the trial is complete, time's up, or the user calls `/ariac/end_competition`